apps/studio/src/lib/db/clients/sqlserver.js

```diff
@@ -76,6 +77,7 @@ export default async function (server, database) {
     selectTop: (table, offset, limit, orderBy, filters, schema, selects) => selectTop(conn, table, offset, limit, orderBy, filters, schema, selects),
     selectTopStream: (db, table, orderBy, filters, chunkSize, schema) => selectTopStream(conn, db, table, orderBy, filters, chunkSize, schema),
     selectTopSql: (table, offset, limit, orderBy, filters, schema, selects) => selectTopSql(conn, table, offset, limit, orderBy, filters, schema, selects),
+    filterSql: (filters) => buildFilterString(filters),
```

Comment:
Did you intend to include this in the exported object?

Reply:
Oops, that's not intended. I was just testing out if I could use the internal `buildFilterString` to put in the raw builder.